node/src/test_utils/neighborhood_test_utils.rs

```diff
@@ -107,8 +69,23 @@ pub fn make_node_record(n: u16, has_ip: bool) -> NodeRecord {
     )
 }
 
-pub fn pick_country_code_record(n: u16) -> (IpAddr, String, bool) {
-    COUNTRY_CODE_DIGEST[n as usize % COUNTRY_CODE_DIGEST.len()].clone()
+pub fn make_node_record_cc(n: u16, has_ip: bool, cc: &str) -> NodeRecord {
```

Comment:
It'd be less code duplication here if you used the original `make_node_record()` to create a mutable result, and then mutated the result with the correct country code.